src/Analysis/Engine/Impl/Interpreter/Ast/AstBuiltinsPythonModule.cs

```diff
@@ -96,13 +95,12 @@ public AstBuiltinsPythonModule(PythonLanguageVersion version)
         }
 
         protected override void PostWalk(PythonWalker walker) {
-            AstPythonBuiltinType boolType = null;
-            AstPythonBuiltinType noneType = null;
+            IPythonType boolType = null;
```

Comment:
This doesn't necessarily have to be IPythonType. `AstPythonType` implements that interface (as the old `AstPythonBuiltinType` did).

Reply:
If they are types, they are `IPythonType`. Unless you prefer specific class?

Reply:
I guess it may not matter. In the previous code, the type was `AstPythonBuiltinType`, which meant in the code below there was nothing other than `biType` to assign to it, which prevents a future code change from making a mistake and assigning the wrong thing to it.

Unless I'm mistaken, now we don't have `AstPythonBuiltinType`, so we don't know if what is coming out of `_members` is a builtin type or not. I'm not sure what the implications of that change are, though.

Reply:
Correct, there is no special 'built-in' class anymore. `IPythonType` provides `IsBuiltin` if type comes from the bultins module. Which is
```cs
bool IsBuiltin => DeclaringModule == null || DeclaringModule is IBuiltinPythonModule
```

Reply:
Yep